### PR TITLE
DVAD-2991: convert drone.yml to 1.0 version

### PR DESCRIPTION
**Pull request by Cotterale**
+ Using "drone convert --save" command
+ Add kubernetes type to drone file

### CODE REVIEW

**Review by rageshrav — DISMISSED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -1,136 +1,188 @@
...
-    event: push
     deploy_to: uat
-    when:
-      branch: master
-      event: push
-      status: success
-  notify_slack_success:
-    image: plugins/slack:1.0
-    secrets: [ slack_webhook ]
+    drone_server: https://drone-gitlab.acp.homeoffice.gov.uk
```
Comment (rageshrav) [inline .drone.yml:134]: new drone server url required here

File: .drone.yml [hunk 2]
```diff
@@ -1,136 +1,188 @@
...
-      - $${kubectl} rollout status "deployment/$${KUBE_DEPLOYMENT_DOCS}"
-    when:
-      branch: master
-      event: push
-  promote_to_uat:
-    image: quay.io/ukhomeofficedigital/drone-trigger:latest
-    secrets: [ drone_token ]
-    drone_server: https://drone-gitlab.acp.homeoffice.gov.uk
-    repo: lev/levops
+    - 0.16.4-${DRONE_BUILD_NUMBER}-${DRONE_COMMIT:0:10}
```
Comment (JamesWoodwardHMPO) [inline .drone.yml:43]: Tags should be in double quotes to avoid some strange drone/yaml behaviour.

File: .drone.yml [hunk 3]
```diff
@@ -1,136 +1,188 @@
...
+    - 0.16.4
+    - 0.16
+    - 0
```
Comment (JamesWoodwardHMPO) [inline .drone.yml:46]: Remove this tag

File: .drone.yml [hunk 4]
```diff
@@ -1,136 +1,188 @@
...
+      from_secret: KUBE_NAMESPACE
+    KUBE_SERVER:
+      from_secret: KUBE_SERVER
+    KUBE_TOKEN:
+      from_secret: KUBE_TOKEN
+    OIDC_URL:
+      from_secret: OIDC_URL
+    PASSWORD:
+      from_secret: PASSWORD
+    TAG: 0.16.4-${DRONE_BUILD_NUMBER}-${DRONE_COMMIT:0:10}
```
Comment (JamesWoodwardHMPO) [inline .drone.yml:91]: Double quotes here too please

**Review by JamesWoodwardHMPO — CHANGES_REQUESTED** [review]
Looks good, with a few minor changes (see comments)

**Review by JamesWoodwardHMPO — APPROVED** [review]

**Pull request merged**